src/components/Letter.tsx

```diff
@@ -81,23 +72,18 @@ export function Letter({ letter, shared, isEditable, disableDrag }: Props) {
       defaultPosition={position}
       onStart={() => {
         setDragging(true);
-        // TODO: update z-index to be the highest.
+        setZIndex(100);
```

Comment:
can you do a real number rather than hard-coding? like keep track of a global Z-index counter for the highest and increment it each time